packages/desktop/src/lib.rs

```diff
@@ -323,6 +323,7 @@ pub fn launch_with_props<P: 'static>(root: Component<P>, props: P, cfg: Config)
                 EventData::Ipc(msg) if msg.method() == "initialize" => {
                     let view = webviews.get_mut(&event.1).unwrap();
                     send_edits(view.dom.rebuild(), &view.desktop_context.webview);
+                    view.desktop_context.webview.window().set_visible(true);
```

Comment:
We should probably respect the visible setting the user sets in the [with_window](https://github.com/DioxusLabs/dioxus/blob/d4e6876ff648159c78ad008ade4e3b1b469bb996/packages/desktop/src/cfg.rs#L98) function in the builder. We could store the visible setting that is set after that function call and only make the window visible again if the visible setting is true

(Invisible windows are useful for headless testing)